### DISCUSSION TITLE
AWS EC2 metadata service

### DISCUSSION THREAD

**Discussion by ktreptow** [concourse/concourse#6977] [Help & Support]
Hello,
Does Concourse need access to the metadata service of the EC2 instances it is running on and is there any reason not to upgrade to IMDSv2? I couldn't find anything related in the documentation. 

Thanks and best regards!

**Comment by taylorsilva** [comment]
Concourse doesn't have anything AWS specific in it besides the AWS secret manager. That uses the official aws-sdk-go package https://github.com/aws/aws-sdk-go/
I assume that package works with IMDS v2 since it is the official one. That repo would be the best place to answer your question though.

**Reply by ktreptow** [reply]
Thanks!
